osidb/api_views.py

```diff
@@ -410,6 +410,11 @@ def create(self, request, *args, **kwargs):
 
 
 class SubFlawViewDestroyMixin:
+    @extend_schema(
+        responses={
+            200: {},
```

Comment:
It can also return 400

Reply:
Well in that case we are missing the 400 and all possible error status codes in every endpoint for every method :laughing: There is a not a single response example for 4xx status code anywhere in the OSIDB currently

Reply:
Fair enough